helpers/tinaHelpers.ts

```diff
@@ -1,11 +1,11 @@
 import { PostDocument, PostFromQuery } from "interfaces";
 
-import { PostQuery } from ".tina/__generated__/types";
-
-const postToDocument = (post: PostFromQuery): PostDocument => {
-  const { date, draft, title, body, heroImage, tags, _sys, id } = post;
+export const postToDocument = (post: PostFromQuery): PostDocument => {
+  const { date, draft, title, body, heroImage, tags, _sys, id, audioVersion } =
+    post;
 
   return {
+    audioVersion: audioVersion ?? "",
```

Comment:
```suggestion
    audioVersion: audioVersion ?? null,
```

Can we do this?

Returning `null` feels like a more proper fallback for the absence of data content

Reply:
Yes, sure we can. Edited